<?xml version="1.0"?>
<launch>
    <arg name="peg_shape" default="cube"/>

    <!--Robot description and related parameter files -->
    <arg name="robot_description_file" default="$(dirname)/inc/load_ur_simple_gripper.launch.xml" doc="Launch file which populates the 'robot_description' parameter."/>
    <arg name="ur_robot" default="ur3e"/>

    <!-- Load urdf on the parameter server -->
    <include file="$(arg robot_description_file)">
        <arg name="ur_robot" value="$(arg ur_robot)"/>
        <arg name="peg_shape" default="$(arg peg_shape)"/>
    </include>

    <!-- <include file="$(find ur_control)/launch/ur_e_controllers.launch">
        <arg name="gripper_robotiq_hande" value="true" />
    </include> -->
    <!-- Spawn robot in gazebo (and move it upwards):
        better initial values should be -2.355 and 1.57 but the robot does
        something strange when unpausing it
    -->
    <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model"
            args="-urdf -param robot_description -model robot 
                -z 0.70 -Y -1.5707"
            output="screen" />

    <rosparam file="$(find ur3_gazebo)/config/gazebo_ros_control_params.yaml" command="load" />

</launch>
